recipes/pyflowy/variants.yaml

```diff
@@ -0,0 +1,5 @@
+c_stdlib_version:
```

Comment:
Unless there is a good reason to specify this, it is best to just use the conda-forge defaults. My suggestion would be to delete this file.